src/network/graphql/queries.js

```diff
@@ -141,7 +140,7 @@ class GraphQuery {
       query: gql`${query}`,
       fetchPolicy: 'network-only',
     });
-    return GraphParser.getParser(this.type)(res.data[this.queryName]);
+    return res.data[this.queryName];
```

Comment:
why we remove the parser?

Reply:
We don't need it. It was just another stupid layer of nothing. We were returning the same exact values as the return. If you want special parsing, do it within the model.